### ISSUE TITLE
QSelect stop working after the last update  v2.1.6

### ISSUE THREAD

**Issue by diadal** [quasarframework/quasar#11042]
`QSelect ` stop working after the last update
```
[Vue warn]: Invalid VNode type: undefined (undefined) 
  at <BaseTransition appear=true persisted=false mode=undefined  ... > 
  at <Transition name="q-transition--fade" appear=true > 
  at <QMenu ref=Ref< Proxy {…} > class="" style=undefined  ... > 
  at <QSelect key=2 behavior="menu" disable=false  ... > 
  at <QForm onSubmit=fn > 
  at <QCard class="my-card-a-1" > 
  at <QPage class="flex flex-center" > 
```

```
» Dev mode.......... spa
 » Pkg quasar........ v2.1.6
 » Pkg @quasar/app... v3.1.5
```

**Comment by MilosPaunovic** [comment]
Have you tried running `quasar clean` command and restarting DEV server, in case something is cached?

**Comment by diadal** [comment]
have downgraded for now i will try your option when done with section am working on

**Comment by rstoenescu** [comment]
@diadal cannot reproduce. can you contact me on Discord for a quick chat pls?

**Comment by rstoenescu** [comment]
Fix will be available in Quasar v2.1.7 ---> just released it as this is a critical bug.
